docs/specs/openapi.yaml

```diff
@@ -0,0 +1,948 @@
+tags:
+- name: OPEN METRICS
+  description: |
+    Exporting of live performance metrics in an Open Metrics compatible format (as described at https://openmetrics.io and popularized by the https://prometheus.io project) is available.
+- name: SCALABLE TIME SERIES
+  description: |
+    The fast, scalable time series query capabilities provided by the [pmseries](https://pcp.io/man/man1/pmseries.1.html)(1) command are also available through a REST API. These queries provide access to performance data (metric metadata and values) from multiple hosts simultaneously, and in a fashion suited to efficient retrieval by any number of web applications.
+
+    All requests in this group can be accompanied by an optional *client* parameter. The value passed in the request will be sent back in the response - all responses will be in JSON object form in this case, with top level "client" and "result" fields.
+- name: FULL TEXT SEARCH
+  description: |
+    The full text search capabilities provided by the [pmsearch](https://pcp.io/man/man1/pmsearch.1.html)(1) command are also available through a REST API. These queries provide access to an index over performance metric names, instances, instance domains and help text, suitable for a metric search engine, and in a fashion suited to efficient querying by any number of web applications.
+
+    In order to use this functionality, the optional *RediSearch* module must be loaded in the **redis-server** at the time **pmproxy** is started, such that metrics, instances and help text it discovers can be automatically indexed.
+- name: PMAPI HOST SERVICES
+  description: |
+    The live performance collection facilities available from [pmcd](https://pcp.io/man/man1/pmcd.1.html)(1) can also be accessed through a REST API.
+
+    All requests are performed on the web server host by default, unless either a *hostspec* or *context* parameter is provided. *hostname* can be used in place of *hostspec*.
+
+    Context identifiers are used as a persistent way to refer to PMAPI contexts across related web requests. These contexts expire after a configurable period of disuse, and are either explicitly allocated using the */pmapi/context* interface, or implicitly allocated using other interfaces.
+
+    The timeout interval is configurable at context creation time, and as such the *polltime* parameter can be used anywhere the *hostspec* is specified. It sets the context timeout in terms of length of inactive time. The unit for the timeout value is seconds and the default is 5.
+
+    To specify a specific existing context in any PMAPI web request, the endpoints can be accessed with either the *context* parameter or enbedded in the endpoint URL itself, such as **/pmapi/[number]/fetch**.
+paths:
+  /metrics:
+    get:
+      tags:
+      - OPEN METRICS
+      summary: Fetches current values and metadata 
+      description: " `Get metrics` fetches current values and metadata for all metrics, or only metrics\
```

Comment:
Can you format this block with `description: |` and normal line breaks (like in the other API methods)?

Reply:
Done!